src/utils/printing.jl

```diff
@@ -1,82 +1,353 @@
-function _display_model(
-    val::Dict{Tuple{String, Symbol}, T},
-    field::Symbol,
-    io::IO,
-) where {T <: ServiceModel}
-    field = titlecase(string(field))
-    println(io, "$(field) Models:\n")
-    if isempty(val)
-        println("\t No Models Specified\n")
-        return
-    end
-    for (i, ix) in val
-        println(
+function Base.show(io::IO, container::OptimizationContainer)
+    show(io, get_jump_model(container))
+end
+
+function Base.show(io::IO, ::MIME"text/plain", input::Union{ServiceModel, DeviceModel})
+    _show_method(io, input, :auto)
+end
+
+function Base.show(io::IO, ::MIME"text/html", input::Union{ServiceModel, DeviceModel})
+    _show_method(io, input, :html)
```

Comment:
Anytime we set the backend to `:html` we need to set `standalone=false` in order to prevent PrettyTables from creating a full HTML document. The default value may be changing in the next release, but we should be explicit. It can cause issues for the package Dheepak is using for documentation.

Reply:
I tried to pass the standalone kwarg and pretty tables threw an error about kwarg not valid

Reply:
I forgot to mention that it requires a newer version of PrettyTables. You probably have 1.1 or 1.2. We bumped the version in PSY and IS to
```
PrettyTables = "^1.3"
```